build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/DependenciesGraphTask.java

```diff
@@ -90,50 +90,49 @@ void generateDependenciesGraph() {
             if (dependency instanceof ProjectDependency) {
                 continue;
             }
-            packageString.append("{\"id\": \"")
-                .append(versionedId)
-                .append("\",\"info\": {\"name\": \"")
-                .append(id)
-                .append("\",\"version\": \"")
-                .append(dependency.getVersion())
-                .append("\"}}");
+            packageString.append("""
+                {"id": "%s","info": {"name": "%s","version": "%s"}}""".formatted(versionedId, id, dependency.getVersion()));
```

Comment:
Do we need a locale here? The docs and the code seem to indicate that this uses the platform default locale. We'd forbid that in regular `String.format`

Reply:
`formatted` doesn't accept a locale :( But AFAIK all calls to formatted use `%s` for string interpolation, so they should be locale independent.